repos/fdbt-site/src/components/MatchingBase.tsx

```diff
@@ -174,14 +178,17 @@ const MatchingBase = ({
     };
 
     if (error) {
-        errors.push({ errorMessage: 'Ensure each fare stage is assigned at least once.', id: 'option-0' });
+        errors.push({ errorMessage: 'No fare stages have been assigned.', id: 'option-0' });
```

Comment:
I think we should have an error message which advises the user on how they can fix their error, whilst also telling them what they have done wrong.

Reply:
So just "No fare stages have been assigned, please assign each fare stage to a stop."

Reply:
Did you fix this @Aaron-Collis-IW ?